meta-lxatac-bsp/recipes-core/images/emmc-image.bb

```diff
@@ -12,9 +12,7 @@ inherit genimage
 COMPATIBLE_MACHINE = "lxatac"
 
 GENIMAGE_IMAGE_SUFFIX = ""
-GENIMAGE_ROOTFS_IMAGE = "lxatac-core-image-base"
-GENIMAGE_ROOTFS_IMAGE_FSTYPE = "tar"
 
 do_genimage[depends] += " \
-    ${GENIMAGE_ROOTFS_IMAGE}:do_image_complete \
+    lxatac-core-image-base:do_image_complete \
 "
```

Comment:
I've changed how the dependency on the rootfs image is handled because `GENIMAGE_ROOTFS_IMAGE` is no longer set.
See the updated PR description for more detail.

Reply:
Looks good.